buildSrc/src/main/kotlin/com/example/util/simpletimetracker/Base.kt

```diff
@@ -6,6 +6,6 @@ object Base {
 
     const val versionCode = 38
     const val versionName = "1.37"
-    const val minSDK = 21
+    const val minSDK = 26
```

Comment:
👀 Here's the `minSDK` bump.

Reply:
I'll see what I can do.

Originally, the `wear` module scaffolded by Android Studio raised the bar to SDK 28 and dropping that to SDK 26 took several painful hours of combing dependency trees to find compatible versions. I eventually reached a point where I considered it more valuable to spend my time building foundational features for the 93% of users on Android >= 8.0 than haggling over build intricacies to attempt to reach that last 7%.

Reply:
> At the moment there were only about a hundred installs from api 21-22, probably just me testing on emulators. For api 23-24-25 it would be about 500 installs

How many of those 500-600 are "Active Devices" as opposed to raw installs? Wondering how many actual people would be impacted by this change if left unresolved.

Reply:
> How many of those 500-600 are "Active Devices"

Probably zero) I'd still prefer minSdk to be as low as humanly possible. As we would need an fdroid build without google play services anyway, it is possible to leave minSdk at 26 for now. Then create separate builds: one with minSdk 21 for fdroid and google play without wear logic, and one with minSdk 26 for google play with wear logic. and then I would see that could be done to lower minSdk for the wear build even lower.

Reply:
Resolving in favor of the more detailed conversation on `app/build.gradle.kts`